core/core.go

```diff
@@ -25,7 +25,7 @@ var (
 	_broadcaster *models.Broadcaster
 )
 
-var handler ffmpeg.HLSHandler
+var handler *ffmpeg.HLSHandler = &ffmpeg.HLSHandler{}
```

Comment:
I had to initialize the pointer here because in [setupStorage](https://github.com/owncast/owncast/blob/master/core/storage.go#L12) it assigns to `handler.Storage`.  I can rework that bit, as it's essentially a no-op as far as I can tell, but I wanted to minimize changes.